build_image.sh

```diff
@@ -32,7 +32,7 @@ else
 	exit 1
 fi
 make clean
-make
+make -j4
```

Comment:
q_: What does this mean?

Reply:
It will launch 4 jobs to run simultaneously which will help to compile the code quickly.

Reply:
I observed quick compilation using `make -j4` than just `make`.

Reply:
@mittachaitu please change this to make -j$(nproc).

Reply:
Updated PTAL